### DISCUSSION TITLE
Using wayland by default on KDE specifically

### DISCUSSION THREAD

**Discussion by RJKAJoaquin** [SpiralLinux/SpiralLinux-project#183] [Q&A]
Hi! I just installed Spiral KDE (testing) on my laptop and I noticed that wayland isn't set by default and isn't even an option on the login screen??? There has already been a question on this for GNOME but since it's GNOME-specific, I'd like to know how to use wayland by default on kde.

Thanks so much!

**Comment by geckolinux** [comment]
Hi there, I don't support or recommend Wayland, but you can follow the normal Debian recommendations for installing and using the Wayland session on your installation.

**Comment by AlphaElwedritsch** [comment]
It's an old thread, but to whom it may concern...

I had the same question. Solution:

`sudo apt install plasma-workspace-wayland` and it is selectable in the KDE login screen
